backend/Dockerfile

```diff
@@ -1,18 +1,22 @@
-FROM ubuntu:16.04
+FROM python:3.8
 
 EXPOSE 5000
 
 ENV LC_ALL=C.UTF-8
 ENV LANG=C.UTF-8
 
-RUN apt-get update -y && apt-get install -y python3-pip libpq-dev && apt-get autoremove
+RUN apt-get update -y && apt-get install -y libpq-dev && apt-get autoremove
 
-RUN pip3 install --upgrade pip
+RUN pip install --upgrade pip
+RUN apt-get update
+
+# update pip
+RUN python -m pip install pip --upgrade
```

Comment:
what's the difference between
```
RUN pip install --upgrade pip
```
and
```
RUN python -m pip install pip --upgrade
```
?

Reply:
As I can see the latter one is recommended for Windows, as there some issues: https://github.com/pypa/pip/issues/1299. I will change it.

Reply:
The linked issue is not affecting us there, because we're using Linux-based container.